app/assets/javascripts/components/stories/History/index.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+import HistoryHeader from './HistoryHeader';
+import HistoryChanges from './HistoryChanges';
+
+const History = ({ history }) => {
+  const array = history.sort(
```

Comment:
You may use [`sortBy`](https://underscorejs.org/#sortBy) here 🤔 
```js
const array = sortBy(history, ({ activity }) => activity.created_at);
```
Check if you will get the same result.
Also, you can extract it to a function:
```js
const sortByActivityCreation = histories => sortBy(histories, ({ activity }) => activity.created_at);
```